app/components/frame-viewer.cjsx

```diff
@@ -33,21 +34,13 @@ module.exports = React.createClass
     {type, format, src} = getSubjectLocation @props.subject, @props.frame
     FrameWrapper = @props.frameWrapper
     zoomEnabled = @props.workflow?.configuration.pan_and_zoom
-    frameDisplay = switch type
+    FileViewer = switch type
       when 'image'
-        <div className="subject-image-frame" >
-          <img ref="subjectImage" className="subject pan-active" src={src} style={SUBJECT_STYLE} onLoad={@handleLoad} tabIndex={0} onFocus={@refs.panZoom?.togglePanOn} onBlur={@refs.panZoom?.togglePanOff}/>
-
-          {if @state.loading
-            <div className="loading-cover" style={@constructor.overlayStyle} >
-              <LoadingIndicator />
-            </div>}
-        </div>
+        ImageViewer
       when 'video'
-        <VideoPlayer src={src} type={type} format={format} frame={@props.frame} onLoad={@handleLoad}>
-        </VideoPlayer>
+        VideoPlayer
       when 'text'
-        <TextViewer  src={src} type={type} format={format} frame={@props.frame} onLoad={@handleLoad} /> 
+        TextViewer 
 
     if FrameWrapper
       <PanZoom ref="panZoom" enabled={zoomEnabled} frameDimensions={@state.frameDimensions}>
```

Comment:
This should be inverted so that `PanZoom` and the SVG image are only rendered for image files.